programmers/예산.js

```diff
@@ -0,0 +1,13 @@
+function solution(d, budget) {
+    let answer = 0;
+    let sum = 0;
+    
+    d.sort((a, b) => (a - b));
+    for (let i = 0; i < d.length; i++){
+        if (sum + d[i] <= budget){
+            sum += d[i];
+            answer++;
+        }
+    }
```

Comment:
for...of 문을 통해서도 풀어볼 수 있음.

```suggestion
    for (let el of d) {
        sum += el;
        if (sum <= budget) answer++;
    }
```

forEach메서드 또는  map메서드로도 가능함

```suggestion
    d.forEach(el => {
        sum += el;
        if(sum <= budget) answer++;
    });
```